### PR TITLE
Update website to callout the issue triage meeting

### LINKED ISSUE #745: Update website community page to mention fortnightly triage meeting
Recently we started a fortnightly triage meeting for the etcd project that re-uses our standard fortnightly community meeting zoom details but just occurs on alternate weeks to the main community meeting.

We should update our website community and triage pages to mention the new meeting:
 - https://etcd.io/community
 - https://etcd.io/docs/v3.6/triage/issues
 - https://etcd.io/docs/v3.5/triage/issues

These files are located under `content/en/community` and `content/en/docs` in the repo, lets update them to advertise and ensure we get as many people along to our fortnightly triage meeting as possible.

### PR DESCRIPTION
**Pull request by mjlshen**
Sibling PR to https://github.com/etcd-io/etcd/pull/16892

Fixes #745

### CODE REVIEW

File: content/en/community/_index.md [hunk 1]
```diff
@@ -65,6 +65,20 @@ channel on [Kubernetes Slack][].
 
 ## Community meetings
 
+etcd contributors and maintainers meet [online][] every every week, on **Thursday
```
Comment (serathius) [inline content/en/community/_index.md:68]:
```suggestion
etcd contributors and maintainers meet [online][] every week, on **Thursday
```

Comment (mjlshen) [reply content/en/community/_index.md]: Thanks - fixed!

File: content/en/community/_index.md [hunk 2]
```diff
@@ -65,6 +65,20 @@ channel on [Kubernetes Slack][].
...
+at 11 AM** [Pacific Time][], alternating between community meetings and issue
+triage meetings. Issue triage meetings are aimed at getting through our backlog
+of PRs and Issues. Triage meetings are open to any contributor; you don't have
+to be a reviewer or approver to help out! They can also be a good way to get
+started contributing.
+
+For phone-in information, the date of the next meeting, and minutes from past
+meetings, see [etcd community meeting][meeting-doc].
+
+{{% cards "community_resources" %}}
```
Comment (jmhbnz) [inline content/en/community/_index.md:78]: Reviewing the deploy preview having two instances of `{{% cards "community_resources" %}}` creates two sections of resources: https://deploy-preview-749--etcd.netlify.app/community

image


The problem is that while the Zoom and YouTube resources are the same for both meetings the Triage meetings use a separate Google Doc which is: https://docs.google.com/document/d/15Plser31Dk5kbWVJufQJvHqNhRLQDrBiZ3nY5NPoTts/edit?usp=sharing

We would need to either:

a) Change to using single google doc, just copy the small content we have in separate doc and bring it into main community doc making it clear which meeting is a triage meeting.

b) Introduce a `{{% cards "triage_resources" %}}` include so we can customize resources for triage.

My vote would go to A to keep things simple.  Keen to leave this open for a couple days to see if anyone has any preferences or other suggestions.

**Review by jmhbnz — CHANGES_REQUESTED** [review]
Overall looks good - we just need to make a decision about resources include duplication.

File: content/en/community/_index.md (hunk 2 above)
Comment (mjlshen) [reply content/en/community/_index.md]: That makes sense to me - I'll update the PR to go in the "A" direction for others to review

**Review by jmhbnz — APPROVED** [review]
LGTM - Thanks @mjlshen

**Review by jberkus — APPROVED** [review]

**Review by serathius — APPROVED** [review]

**Pull request merged**